api/Animation.json

```diff
@@ -25,7 +25,7 @@
             }
           ],
           "edge": {
-            "version_added": "≤79"
+            "version_added": "79"
```

Comment:
I think Florian and co. were using this value because they didn't know whether this was supported in non-Chromium Edge. What do you know that I don't?

Reply:
As mentioned above, it was found that neither interface was supported in pre-Chrome Edge versions.￼￼

Reply:
Indeed, I tested it and it's not there. The "test" is simply logging the expressions `Animation` and `KeyframeEffect`, so I didn't link to a test in the description.